src/Resources/config/resources.yml

```diff
@@ -3,6 +3,7 @@ sylius_resource:
         solutiondrive.product_bundle:
             classes:
                 model: SolutionDrive\SyliusProductBundlesPlugin\Entity\ProductBundle
+                form: SolutionDrive\SyliusProductBundlesPlugin\Form\Type\ProductBundleType
```

Comment:
In my opinion we should switch in this plugin the base namespace to `solutionDrive` as defined by corporate identity.

Reply:
I created an issue for this:
#19